Makefile

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 #
 
-VERSION ?= latest
+VERSION ?= 0.1.0
```

Comment:
I think we should set it to `0.2.0-latest`? And with a real release script to override this?

In traditional, we should follow the pattern, version + snapshot/latest/commit id as dev version. 0.1.0， 0.2.0 should only work for release, right?

Reply:
> I think we should set it to 0.2.0-latest?

> In traditional, we should follow the pattern, version + snapshot/latest/commit id as dev version. 0.1.0， 0.2.0 should only work for release, right?

I prefer `0.2.0-snapshot`

> And with a real release script to override this?

Now the `makefile` serves as release script, if we add a release script, then the version in that script should be updated in every release? say `sh release.sh --version=0.2.0`, I don't think it's different from `make VERSION=0.2.0 release`.

Reply:
My point is, when it is not release tag, it should package the codes as 0.2.0-snapshot. When the release happens, people should input
1. GitHub user name
1. Github password
1. Tag name
1. Release version
1. Next version

Then all things should be changed, and tag after those, push tag and git commits to the branch. Then move the version to the next version. So there are 3 git push
1. Version change to stable version(no snapshot) as commit 1
1. Tag this commit and push
1. Version changes to the next snapshot as commit 2

You should have used this in the main repo release, right? This is my expected release script. Make sense?

Reply:
Uh I got your point now, you're expecting the release process to be more automated, make sense to me absolutely

Reply:
> Uh I got your point now, you're expecting the release process to be more automated, make sense to me absolutely

Yes. I hope more people are able to be the release manager. As this project is not like the main repo, people are facing the network issue.